src/ajax/check_hgvs_dialogue.php

```diff
@@ -0,0 +1,603 @@
+        update_dialogue(
+            '<br>We could not validate nor map your variant because of the following problem(s):<br>- ' .
+            implode('<br> -', $aMappedVariant['errors']) . '<br><br>' .
+            'Please take another look at your variant and try again.',
+            $sButtonOKInvalid
+        );
+        exit;
+    }
+
+    // Check for warnings.
+    if (!empty($aMappedVariant['warnings'])) {
+        // One or more warnings were found. Perhaps the variant was corrected?
+
+        if (isset($aMappedVariant['warnings']['WROLLBACK'])
+            || isset($aMappedVariant['warnings']['WCORRECTED'])) {
+            // The variant was corrected.
+            update_dialogue('<br>Your variant was corrected to ' . $aMappedVariant['data']['DNA'] .
+                ' to fully match HGVS guidelines.');
+            $bImprovedByVV = true;
+        }
+
+        if (isset($aMappedVariant['warnings']['WFLAG'])) {
+            // This type of warning tells us that VariantValidator had a problem
+            //  which is an issue with them, not us nor our user. We can only get
+            //  the mapping on all genome builds, not on (other) transcripts.
+            // We will notify the user.
+            update_dialogue('<br>Your variant could not fully be validated due to unknown issues.');
+            // Fixme; Either find a fix within VV, or Call Mutalyzer.
+        }
+
+    // Check whether the mapping was successful.
+    } elseif (!isset($aMappedVariant['data']['DNA'])
+        || empty($aMappedVariant['data']['DNA'])) {
+        // Although we did not receive any warnings or errors, the DNA field
+        //  is left empty. This means we have no information on the mapping
+        //  and there is not much we can do... We will inform the user that
+        //  an unknown error occurred and that they should try again later.
+        update_images_per_step($sStepMapping, $sImageFailed);
+        update_dialogue(
+            '<br>An unknown error occurred while trying to validate and map your variant.' .
+            ' We are sorry for the inconvenience. Please try again later.',
+            $sButtonOKInvalid
+        );
+        exit;
+    }
+
+
+    // When sending in a variant on transcript, VariantValidator only
+    //  returns the variant as mapped on that one transcript. If we are
+    //  on a VOT creation form, and there are multiple transcripts open,
+    //  we want each transcript to get a mapping. To get this, we then
+    //  need to call VariantValidator a second time using one of the
+    //  genomic variants as were returned using our first call.
+    if ($sType == 'VOT' && count($aTranscripts) > 1) {
+
+        $aMappedViaGB = (
+        !isset($aMappedVariant['data']['genomic_mappings']['hg38']) ? // Yes=We have a genomic reference from our first call; No=We don't have a genomic reference.
+            array() :
+            $_VV->verifyGenomic($aMappedVariant['data']['genomic_mappings']['hg38'], array(
```

Comment:
I don't feel comfortable having a certain genome build hardcoded into the source. How can we always pick the latest build? I think we can safely assume;
- The order of builds in `$_SETT['human_builds']` is from old to new.
- We won't add new builds unless VV supports them.